cmd/podman/pods/stats.go

```diff
@@ -0,0 +1,189 @@
+package pods
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+	"text/tabwriter"
+	"text/template"
+	"time"
+
+	"github.com/buger/goterm"
+	"github.com/containers/buildah/pkg/formats"
+	"github.com/containers/libpod/cmd/podman/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/util/camelcase"
+	"github.com/spf13/cobra"
+)
+
+type podStatsOptionsWrapper struct {
+	entities.PodStatsOptions
+
+	// Format - pretty-print to JSON or a go template.
+	Format string
+	// NoReset - do not reset the screen when streaming.
+	NoReset bool
+	// NoStream - do not stream stats but write them once.
+	NoStream bool
+}
+
+var (
+	statsOptions     = podStatsOptionsWrapper{}
+	statsDescription = `For each specified pod this command will display percentage of CPU, memory, network I/O, block I/O and PIDs for containers in one the pods.`
```

Comment:
I'm confused, and this might be old verbiage.  You can ask for stats on 1 to n pods, but the output is for the containers in only one of the pods?   Maybe "for containers in one of the pods" -> "for the containers within each specified pod"?

Reply:
Down below we note this works only on "running" pods.  We should note this up here too.